### PR TITLE
added userdto, configured webapp

### PR DESCRIPTION
**Pull request by tarkaiev**

### CODE REVIEW

File: src/main/java/tutorial/spring/controllers/UserController.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import tutorial.spring.dto.UserResponseDto;
+import tutorial.spring.model.User;
+import tutorial.spring.service.UserService;
+
+@RestController
+@RequestMapping("/user")
+public class UserController {
+
```
Comment (AnaLanda) [inline src/main/java/tutorial/spring/controllers/UserController.java:17]: I think you can remove this empty line)

Comment (AnaLanda) [inline src/main/java/tutorial/spring/controllers/UserController.java:17]: I think you can remove the empty line here

**Review by AnaLanda — COMMENTED** [review]
Looks great! 😃

File: src/main/java/tutorial/spring/config/WebConfig.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import org.springframework.web.servlet.view.InternalResourceViewResolver;
+import org.springframework.web.servlet.view.JstlView;
+
+@Configuration
+@EnableWebMvc
+@ComponentScan(basePackages = {"tutorial.spring.controllers"})
+public class WebConfig {
+
+    @Bean
+    public InternalResourceViewResolver resolver() {
```
Comment (Sofasmile) [inline src/main/java/tutorial/spring/config/WebConfig.java]: let's remove this bean

File: src/main/java/tutorial/spring/controllers/UserController.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    @GetMapping
+    public List<UserResponseDto> getAll() {
+        return userService.listUsers().stream()
+                .map(this::getUserRespDto)
+                .collect(Collectors.toList());
+    }
+
+    @GetMapping("/{userId}")
+    public UserResponseDto get(@PathVariable Long userId) {
+
```
Comment (Sofasmile) [inline src/main/java/tutorial/spring/controllers/UserController.java]: remove empty line

File: src/main/java/tutorial/spring/controllers/UserController.java [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+        return getUserRespDto(userService.get(userId));
+    }
+
+    private UserResponseDto getUserRespDto(User user) {
```
Comment (Sofasmile) [inline src/main/java/tutorial/spring/controllers/UserController.java]:
```suggestion
    private UserResponseDto getUserResponseDto(User user) {
```

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request merged**